apps/edxapp/vars/all/main.yml

```
@@ -45,12 +45,14 @@ edxapp_mongodb_image_name: "centos/mongodb-32-centos7"
 edxapp_mongodb_image_tag: "3.2"
 edxapp_mongodb_port: 27017
 edxapp_mongodb_secret_name: "edxapp-mongo-{{ edxapp_vault_checksum }}"
+edxapp_mongodb_version: "3.2"
 
 # -- mysql
 edxapp_mysql_image_name: "centos/mysql-57-centos7"
 edxapp_mysql_image_tag: "5.7"
 edxapp_mysql_port: 3306
 edxapp_mysql_secret_name: "edxapp-mysql-{{ edxapp_vault_checksum }}"
+edxapp_mysql_version: "5.7"
```

Same suggestion here:

```suggestion
edxapp_mysql_version: "5.7"
edxapp_mysql_image_name: "centos/mysql-57-centos7"
edxapp_mysql_image_tag: "{{ edxapp_mysql_version }}"
```